scripts/build/utils/build-theme.ts

```diff
@@ -0,0 +1,31 @@
+import * as fs from "fs";
+import * as path from "path";
```

Comment:
To be consistent with
https://github.com/cadamsdev/vscode-jetbrains-icon-theme/pull/103/files#diff-9f1ca9806327e5685f7a6e6528cfa7ccea6b043cc3bde415cdffec38ccfbbec3R1

```suggestion
import * as path from "node:path";
```